apps/email/style/setup_cards.css

```diff
@@ -354,27 +349,30 @@ li.align-center {
 }
 
 .item-with-children::after  {
-  content: "";
   width: 2.6rem;
   height: 2.6rem;
   position: absolute;
   top: calc(50% - 1.3rem);
-  right: -0.5rem;
-  background: url('../shared/style/buttons/images/forward-light.svg') no-repeat center / 2.6rem;
+  offset-inline-end: -0.5rem;
+
+  font-family: 'gaia-icons';
+  font-weight: 500;
+  font-size: 3rem;
+  font-style: normal;
+  text-decoration: inherit;
+  text-transform: none;
+  text-rendering: optimizeLegibility;
   opacity: 0.7;
 }
-
-html:-moz-dir(rtl) .item-with-children::after  {
-  left: -0.5rem;
-  right: auto;
-}
+html[dir="ltr"] .item-with-children::after  { content: 'right-light'; }
+html[dir="rtl"] .item-with-children::after  { content: 'left-light'; }
```

Comment:
If you agree with what I've said about gaia-icons, we could have a 

```css
.icon-after::after {
  content: attr('data-icon-after');
  font-family: 'gaia-icons';
  font-weight: 500;
  font-size: 3rem;
  font-style: normal;
  text-decoration: inherit;
  text-transform: none;
  text-rendering: optimizeLegibility;
}
```
Or something like that, with or without `width`, `height`, `position`... it depends if everything uses the same values.

And in the html, we would just need to put a `data-icon-after="myIconName"` on the element to give it an icon.